include/GLFW/glfw3.h

```diff
@@ -3875,9 +3877,14 @@ GLFWAPI int glfwGetInputMode(GLFWwindow* window, int mode);
  *  GLFW_MOD_CAPS_LOCK bit set when the event was generated with Caps Lock on,
  *  and the @ref GLFW_MOD_NUM_LOCK bit when Num Lock was on.
  *
+ *  If the mode is `GLFW_RAW_INPUT`, the value must be either `GLFW_TRUE` to
+ *  enable the use of raw input, or `GLFW_FALSE` to disable it.  If enabled and
```

Comment:
You have a double space between "disable it.__If"

Reply:
Yes, I saw that everywhere else there were two spaces after dots, so I did the same.